.github/workflows/release.yml

```diff
@@ -71,9 +71,10 @@ jobs:
     needs: [build, release]
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3.1.0
+      - uses: actions/checkout@v4
         with:
           repository: 'GreptimeTeam/greptimedb'
+          token: ${{ secrets.PR_ACTION }}
```

Comment:
Is this token required here?

Reply:
It should be the token of line 83. I doubt if the token has expired.

Reply:
So I'd revert this change line and you check the secrets or replace it with a new generated PAT?

Reply:
I need to confirm with @ZonaHex if she has the token

Reply:
Well. Let me revert this change line and we can review this topic later. We can still merge the other changes >_<